### ISSUE TITLE
Nav: "COVE Editions" link should be dead

### ISSUE THREAD

**Issue by jamiefolsom** [SHOREdevelopers/COVE#57]
At https://editions.covecollective.org/, the white link in the black nav ribbon which reads "COVE Editions" should have an href attr val of "#".

**Comment by jamiefolsom** [comment]
@daverett ok if I put some issues in here for you? Just want to document them at least, so we can track 'em. Thx.

**Comment by daverett** [comment]
yep, @jamiefolsom. Thanks. Just fixed this one via https://github.com/SHOREdevelopers/COVE/commit/41b9cd44f3bc8c4e5c36d089a8fdd3d173734715
